src/settings.rs

```diff
@@ -10,7 +10,8 @@ use std::path::{Path, PathBuf};
 #[derive(Deserialize, Debug, Clone)]
 pub struct Policy {
     pub url: String,
-
+    #[serde(default)] // default bool value is "false"
+    pub allowed_to_mutate: bool,
```

Comment:
I think the plan was to have something like:

```suggestion
    #[serde(default)]
    pub allowed_to_mutate: Option<bool>,
```

If `None` is provided (it wasn't specified in the config file explicitly), then default `allowed_to_mutate` to the policy metadata mutating field.